tests/functional_tests/ffront_tests/test_execution.py

```diff
@@ -727,3 +727,63 @@ def reduce_tuple_element(
     expected = red[rs.v2e_table][:, 0]
 
     assert np.allclose(expected, out)
+
+
+def test_tuple_program_return_constructed_inside(fieldview_backend):
```

Comment:
The purpose of these tests is to test the Program Part so I would place them in `test_program.py`. The field operator part of these tests is already tested above. Otherwise the `test_program.py` is superfluous and should be integrated here.

Reply:
moved